src/current/_includes/releases/v23.1/v23.1.19.md

```diff
@@ -0,0 +1,10 @@
+## v23.1.19
+
+Release Date: April 18, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-19-bug-fixes">Bug fixes</h3>
+
+- Reintroduced [cluster setting]({% link v23.1/cluster-settings.md %}) `sql.auth.modify_cluster_setting_applies_to_all.enabled` so that mixed-version clusters can migrate off of this setting. It is deprecated in favor of the role [`MODIFYSQLCLUSTERSETTING`]({% link v23.1/set-cluster-setting.md %}#required-privileges). [#122455][#122455]
```

Comment:
```suggestion
- Reintroduced the [cluster setting]({% link v23.1/cluster-settings.md %}) `sql.auth.modify_cluster_setting_applies_to_all.enabled` so that mixed-version clusters can migrate off of this setting, which is deprecated in favor of the role [`MODIFYSQLCLUSTERSETTING`]({% link v23.1/set-cluster-setting.md %}#required-privileges). [#122455][#122455]
```
I think this makes the call to action to get off the setting a bit more clear. WDYT?

Reply:
FWIW, I like this edit.

Reply:
> It is deprecated in favor of the role MODIFYSQLCLUSTERSETTING

this should instead say "It is deprecated in favor of the privilege MODIFYSQLCLUSTERSETTING"

(to say more, privileges are the things that let you do an action. privileges are granted _to_ roles or users)

Reply:
Thanks, all. I've updated this in accordance with your feedback.